src/org/zaproxy/zap/extension/dynssl/ExtensionDynSSL.java

```diff
@@ -146,4 +153,54 @@ public Certificate getRootCA() throws KeyStoreException {
 	public boolean supportsDb(String type) {
     	return true;
     }
+	
+	/**
+	 * Gets ZAPs current CA Certificate in X.509 format.
+	 * 
+	 * @return The X.509 version of ZAPs current CA certificate.
+	 */
+	public X509Certificate getCaCertificate() {

```

Comment:
- The method name could have "RootCa".
 - Worth adding the since tag (same for the other methods that stay public).